### ISSUE TITLE
PHP Warning:  Undefined array key "FirstChar" in Smalot/PdfParser/Font.php on line 291

### ISSUE THREAD

**Issue by mbideau-atreal** [smalot/pdfparser#731]
Hi,

Same with `LastChar`:
```
PHP Warning:  Undefined array key "LastChar" in Smalot/PdfParser/Font.php on line 291
```

I am sorry I can't provide a sample PDF cause it is classified document.

The line is: https://github.com/smalot/pdfparser/blob/master/src/Smalot/PdfParser/Font.php#L291
The version I use is the 2.10: https://github.com/smalot/pdfparser/blob/v2.10.0/src/Smalot/PdfParser/Font.php#L291

Thanks in advance for solving this bug.

Best regards

**Comment by TilsonM17** [comment]
Same problem. Any solution?

**Comment by k00ni** [comment]
@TilsonM17 can you provide a PDF file which triggers the error?

**Comment by TilsonM17** [comment]
@k00ni yes I can. This PDF have fake data, the real PDF have sensitive information 's client. But is the same pdf layout

[exe01.pdf](https://github.com/user-attachments/files/16694622/exe01.pdf)

**Comment by mbideau-atreal** [comment]
Hi, any progress on this.
I can help testing if needed.
Thanks in advance.
Best regards
